MIDIchlorians/MIDIchlorians/AnimationDesignerController.swift

```diff
@@ -154,3 +154,10 @@ class AnimationDesignerController: UIViewController {
         delegate?.saveAnimation()
     }
 }
+
+extension AnimationDesignerController: PadDelegate {
+    func pad(animationUpdated animation: AnimationSequence) {
+        print(animation.animationBitsArray)
+        timelineView.frames = animation.animationBitsArray.map { ($0?.count ?? 0) > 0 }
```

Comment:
Yea i think it will be nice to put it in animation sequence, not too important, we can get to it next time :)